pandas/io/common.py

```diff
@@ -150,38 +137,20 @@ def urlopen(*args, **kwargs):
     return urllib.request.urlopen(*args, **kwargs)
 
 
-def get_fs_for_path(filepath: str):
+def is_fsspec_url(url: FilePathOrBuffer) -> bool:
     """
-    Get appropriate filesystem given a filepath.
-    Supports s3fs, gcs and local file system.
-
-    Parameters
-    ----------
-    filepath : str
-        File path. e.g s3://bucket/object, /local/path, gcs://pandas/obj
-
-    Returns
-    -------
-    s3fs.S3FileSystem, gcsfs.GCSFileSystem, None
-        Appropriate FileSystem to use. None for local filesystem.
+    Returns true if fsspec is installed and the given URL looks like
+    something fsspec can handle
```

Comment:
Hence "looks like"; whether it can actually be handled will depend on fsspec and whichever backend being installed - but you'll get the error shortly after, when trying to use the URL.